utils/ort/src/main/kotlin/Utils.kt

```diff
@@ -123,7 +124,21 @@ fun filterVersionNames(version: String, names: List<String>, project: String? =
  * Request a [PasswordAuthentication] object for the given [host], [port], and [scheme]. Install the [OrtAuthenticator]
  * and the [OrtProxySelector] beforehand to ensure they are active.
  */
-fun requestPasswordAuthentication(host: String, port: Int, scheme: String): PasswordAuthentication? {
+fun requestPasswordAuthentication(host: String, port: Int, scheme: String): PasswordAuthentication? =
+    queryAuthenticator(host, port, scheme, null)
```

Comment:
This looks a bit "Java-style". Why not achieve the same via a default `url: URL? = null` argument below?